src/plugins/contextMenu/predefinedItems/rowAbove.js

```diff
@@ -24,7 +24,7 @@ export default function rowAboveItem() {
         return true;
       }
 
-      return this.selection.isSelectedByColumnHeader() || this.countRows() >= this.getSettings().maxRows;
+      return this.selection.isSelectedByColumnHeader() || this.selection.isSelectedByCorner() || this.countRows() >= this.getSettings().maxRows;
```

Comment:
The line exceeds 120 characters. Additionally, I didn't found any test about disability the "Insert row above" option in the context menu when the corner is clicked. Can I ask you to fix that?

Reply:
Same as https://github.com/handsontable/handsontable/pull/6969#discussion_r434445094